environment.yml

```diff
@@ -0,0 +1,66 @@
+name: cognoma-cancer-data
+dependencies:
```

Comment:
@dhimmel It seems like it is the same issue with anaconda channel availability for the other two as well:
[`libsodium`](https://anaconda.org/anaconda/libsodium/files) - libsodium 1.0.3 is the most recent version available on linux/windows/osx
[`fontconfig`](https://anaconda.org/anaconda/fontconfig/files) - only available for linux? not sure what to do about this